simplified-viewer-epub-readium2/src/main/java/org/nypl/simplified/viewer/epub/readium2/Reader2Activity.kt

```diff
@@ -302,7 +302,7 @@ class Reader2Activity : AppCompatActivity(R.layout.reader2) {
 
     return when (event) {
       SR2ReaderViewNavigationClose ->
-        this.tocClose()
+        this.onBackPressed()
```

Comment:
This is a one line change. The existing `onBackPressed` method calls `this.tocClose()` if the TOC is open, or the superclass `onBackPressed` otherwise. The latter will pop back the stack.

https://github.com/ThePalaceProject/android-core/blob/9b839bd0460060718c29522bd4810cdec2cd84e3/simplified-viewer-epub-readium2/src/main/java/org/nypl/simplified/viewer/epub/readium2/Reader2Activity.kt#L344-L350